src/main/java/fi/foyt/foursquare/api/NotificationsParser.java

```diff
@@ -81,6 +81,12 @@ private NotificationsParser() {
               ScoreNotification scoresNotification = (ScoreNotification) JSONFieldParser.parseEntity(ScoreNotification.class, item, skipNonExistingFields);
               result.add(new Notification<ScoreNotification>(notificationType, scoresNotification));
             break;
+            case NotificationTray:
+              // TODO
```

Comment:
Is this just a placeholder for these fields, and the parsing wold still need to be done?